test/bdd/db/query/japanese.feature

```diff
@@ -0,0 +1,29 @@
+@DB
+Feature: Searches in Japan
+    Test specifically for searches of Japanese addresses and in Japanese language.
+    Scenario: A block house-number is parented to the neighbourhood
```

Comment:
Given that your test needs a sanitizer, it is natural that it cannot work with the legacy tokenizer system that does not know about sanitizers.  Simply add a `@fail-legacy` above the scenario.

Reply:
Thank you so much. I understood.
I added @fail-legacy.